Scripts/gen_Primary.sh

```diff
@@ -195,17 +178,26 @@ if [[ "${FLAT}" == "FlatMuDaughter" ]]; then
   echo physics.producers.generate.endMom: ${ENDMOM}        >> primary.fcl
 fi
 
-#
-# now generate the fcl
-#
+if [[ -n $SETUP ]]; then
+  echo "Using user-provided setup $SETUP"
+else
+  SETUP=/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/${PRIMARY_CAMPAIGN}/setup.sh
+fi
 
-generate_fcl --dsconf=${PRIMARY_CAMPAIGN} --dsowner=${OWNER} --run-number=${RUN} --description=${PRIMARY} --events-per-job=${EVENTS} --njobs=${JOBS} \
-  --embed primary.fcl --auxinput=1:physics.filters.${resampler}.fileNames:Stops.txt
-for dirname in 000 001 002 003 004 005 006 007 008 009; do
-  if test -d $dirname; then
-    echo "found dir $dirname"
-    rm -rf ${PRIMARY}\_$dirname
-    mv $dirname ${PRIMARY}\_$dirname
-    echo "moving $dirname to ${PRIMARY}_${dirname}"
-  fi
-done
+cmd="mu2ejobdef --embed primary.fcl --setup ${SETUP} --run-number=${RUN} --events-per-job=${EVENTS} --desc ${PRIMARY} --dsconf ${PRIMARY_CAMPAIGN} --auxinput=1:physics.filters.${resampler}.fileNames:Stops.txt"
```

Comment:
mu2ejobdef assumes you are running in a unique directory: the output tar file is always produced in the workdir, and it will combine the results from any previous invocation of mu2ejobdef. I suggest to delete any previous .tar files in this directory to avoid confusion, at least in production mode. Ideally mu2ejobdef would take care of this itself: I have suggested it to the author but did not receive a positive response.

Reply:
Outside the scope of this PR: Do we want to consider archiving the tar file in 'production' mode? Maybe making a copy on persistent?

Reply:
>I suggest to delete any previous .tar files in this directory to avoid confusion
Will do